### DISCUSSION TITLE
trouble in paradise (i.e cnn doesn't seem to be working )

### DISCUSSION THREAD

**Discussion by Skruller01** [mrdbourke/pytorch-deep-learning#969] [General]
in the 3rd section i.e the computer vision i am facing trouble in cnn part , the model  should be showing atleast 88% accuracy but actually showing 8% accuracy to be me ,there is some sort of fixing in the code that is required but i can't seem to get it. Can somebody help me with this?
![Screenshot 2024-06-14 011423](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/04a2e13d-9fbf-4929-9f34-719b1595771d)
![Screenshot 2024-06-14 011433](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/66aefce6-719e-4825-a13f-3de953ed8c55)
![Screenshot 2024-06-14 011446](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/1ab48cbc-2138-4814-8c8b-30a77a965bc2)
![Screenshot 2024-06-14 011501](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/c2247e1d-cf19-4dd8-a2ee-87643bc595b8)
![Screenshot 2024-06-14 011512](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/39e321d1-97bb-483d-998c-0cfb62325728)
![Screenshot 2024-06-14 011525](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/d1f168c7-8643-4d6c-9819-3ae4bfba9af2)
![Screenshot 2024-06-14 011533](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/c9c9a256-7897-42e4-9051-f67385eca9cf)
![Screenshot 2024-06-14 011541](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/072673cc-a88d-4dbd-a025-b1ff4dc5dbb5)
![Screenshot 2024-06-14 011551](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/5fe94c9b-ebb2-4cca-ab17-697e4e6f3761)
![Screenshot 2024-06-14 011559](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/26c052ef-72c7-48fc-8915-a22ceb94cddc)
![Screenshot 2024-06-14 011610](https://github.com/mrdbourke/pytorch-deep-learning/assets/115204364/ca824c3c-21fe-4905-aadc-f8c9630c7548)

**Comment by LuluW8071** [comment]
Try increasing learning rate to `0.01` or `0.001`.
